brody_lab_to_nwb/msortedsortinginterface.py

```diff
@@ -0,0 +1,39 @@
+"""Authors: Cody Baker."""
+import numpy as np
+
+from h5py import File
+from nwb_conversion_tools.datainterfaces.ecephys.basesortingextractorinterface import BaseSortingExtractorInterface
+
+from .msortedsortingextractor import MSortedSortingExtractor
+
+
+class MSortedSortingInterface(BaseSortingExtractorInterface):
+    """Conversion class for the pre-sorted data corresponding to the Neuralynx format for the Brody lab."""
+
+    SX = MSortedSortingExtractor
+
+    @classmethod
+    def get_source_schema(cls):
+        source_schema = dict(
+            required=["file_path"],
+            properties=dict(
+                file_path=dict(
+                    type="string",
+                    format="file",
+                    description="Path to .mat file containing processed data."
+                )
+            ),
+            type="object",
+            additionalProperties=False
+        )
+        return source_schema
```

Comment:
can we use `super` here?

Reply:
That would depend sensitively on the nwb-conversion-tools version. After PR #237 is merged on nwb-conversion-tools and the version updated here, it likely could. For now, on the version for this PR, I can guarantee this will run without error.